File: Wildcat All-In-One Timer/UnhandledExceptionForm.cs
using System;
using System.Drawing;
using System.Windows.Forms;

namespace UTimer
{
    public partial class UnhandledExceptionForm
    {
        public UnhandledExceptionForm()
        {
            InitializeComponent();
            _ExceptionLabel.Name = "ExceptionLabel";
            _ExitButton.Name = "ExitButton";
        }

        public object exce;

        private void ExitButton_Click(object sender, EventArgs e)
        {
            Application.Exit();
        }

        private void UnhandledExceptionForm_Load(object sender, EventArgs e)
        {
            ExceptionLabel.Text = "Click to see exception details.";
            BackColor = ColorTranslator.FromHtml(My.MySettingsProperty.Settings.BackColor.ToArgb().ToString());
            ForeColor = ColorTranslator.FromHtml(My.MySettingsProperty.Settings.ForeColor.ToArgb().ToString());
        }

        private void ExceptionLabel_Click(object sender, EventArgs e)
        {
            MessageBox.Show(exce.Message.ToString(), "Kill Code " + exce.hresult.ToString(), MessageBoxButtons.OK, MessageBoxIcon.Error);
        }
    }
}

File: Wildcat All-In-One Timer/UnhandledExceptionForm.vb
Option Strict Off
Public Class UnhandledExceptionForm
    Public exce
    Private Sub ExitButton_Click(sender As Object, e As EventArgs) Handles ExitButton.Click
        System.Windows.Forms.Application.Exit()
    End Sub

    Private Sub UnhandledExceptionForm_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        ExceptionLabel.Text = "Click to see exception details."
        Me.BackColor = System.Drawing.ColorTranslator.FromHtml(CStr(My.Settings.BackColor.ToArgb))
        Me.ForeColor = System.Drawing.ColorTranslator.FromHtml(CStr(My.Settings.ForeColor.ToArgb))
    End Sub

    Private Sub ExceptionLabel_Click(sender As Object, e As EventArgs) Handles ExceptionLabel.Click
        MessageBox.Show(exce.Message.ToString, "Kill Code " & exce.hresult.ToString, MessageBoxButtons.OK, MessageBoxIcon.Error)
    End Sub
End Class